docker/manager.py

```diff
@@ -115,12 +115,14 @@ def read_file(self, path):
         :type path: str
         :return: The content of the file
         :rtype: str
+        :raises DockerFileNotFoundError: If given an invalid path
         """
         path = self._get_working_directory(path)
         result = self.run('cat {0}'.format(path))
-        if result.succeeded:
-            return result.out
-        return None
+        if not result.succeeded:
```

Comment:
Maybe `result.err` should be checked to make sure it's actually a file not found error? Then it needs to raise something else if it's not though.